tests/unit/workspace_access/test_migration_state.py

```diff
@@ -0,0 +1,19 @@
+from databricks.labs.ucx.workspace_access.groups import MigratedGroup, MigrationState
```

Comment:
by the naming logic, this test should be in the `test_groups`, not the `test_migration_state`.

Reply:
moved to test_groups